server/api/eliteApi.test.js

```diff
@@ -0,0 +1,71 @@
+const nock = require('nock')
+const MockAdapter = require('axios-mock-adapter')
+
+const clientFactory = require('./oauthEnabledClient')
+const { eliteApiFactory } = require('./eliteApi')
+const contextProperties = require('../contextProperties')
+
+const hostname = 'http://localhost:8080'
```

Comment:
@JamieDraperUK not sure how I managed to remove your comment regarding matching config.js

Anyway, no.  Could be anything.  However extracted out as nock requires just the hostname (see https://github.com/nock/nock#specifying-hostname) and not the trailing slash